modules/va_facilities/README.yml

```diff
@@ -424,6 +424,197 @@ paths:
+          in: query
+          description: |
+            Street address of the location from which drive time will be calculated.
+          required: false
+          schema:
+            type: string
+        - name: city
+          in: query
+          description: |
+            City of the location from which drive time will be calculated.
+          required: false
+          schema:
+            type: string
+        - name: state
+          in: query
+          description: |
+            Two character state code of the location from which drive time will be calculated.
+          required: false
+          schema:
+            type: string
+        - name: zip
+          in: query
+          description: |
+            Zip code of the location from which drive time will be calculated.
+          required: false
+          schema:
+            type: string
+            format: '##### or #####-####'
+        - name: lat
+          in: query
+          description: |
+            Latitude of point to search for nearest VA Facilities.
+          required: false
+          schema:
+            type: number
+            format: float
+        - name: lng
+          in: query
+          description: |
+            Longitude of point to search for nearest VA Facilities.
+          required: false
+          style: form
+          explode: true
+          schema:
+            type: number
+            format: float
+        - name: drive_time
+          description:  Filter to only include facilities that are within the specified number of drive time minutes from the requested location.
+          in: query
+          required: false
+          schema:
+            type: integer
+            default: 90
+            enum:
+              - 10
+              - 20
+              - 30
+              - 40
+              - 50
+              - 60
+              - 70
+              - 80
+              - 90
+        - name: services[]
+          description: Optional facility service search filter
+          in: query
+          style: form
+          explode: true
+          schema:
+            type: array
+            items:
+              type: string
+        - name: page
+          description: Page of results to return per paginated response.
+          in: query
+          required: false
+          schema:
+            type: integer
+            default: 1
+        - name: per_page
+          description: Number of results to return per paginated response.
+          in: query
+          required: false
+          schema:
+            type: integer
+            default: 30
+      responses:
+        '200':
+          description: Success
+          content:
+            application/json:
+              schema:
+                required:
+                  - data
+                properties:
+                  data:
+                    type: array
+                    items:
+                      $ref: "#/components/schemas/NearbyFacility"
```

Comment:
The `NearbyFacility` schema object this references needs to be included as well.

Reply:
editor.swagger.io didn't show the error until refresh 🤦‍♂